src/components/20-molecules/accordion/index.js

```diff
@@ -35,100 +77,83 @@ class AXAAccordion extends LitElement {
+    if (this.open) {
+      collapseContent(accordionContent);
+    } else {
+      expandContent(accordionContent);
+    }
+
+    this.open = !this.open;
+    this.onStateChange(this.open);
+    fireCustomEvent('axa-state-change', this.open, this);
   }
 
   render() {
     const { open, title, small, disabled, icon } = this;
     const titleContainerClasses = {
-      'm-accordion__title-container': true,
-      'm-accordion__title-container--disabled': disabled,
+      'm-accordion__header': true,
+      'm-accordion__header--disabled': disabled,
     };
     const titleClasses = {
       'm-accordion__title': true,
       'm-accordion__title--small': small,
     };
 
-    const iconClass = icon
-      ? 'm-accordion__title-container-title-icon'
-      : 'm-accordion__title-container-title-icon--hidden';
-
-    const statusIcon = unsafeHTML(open ? expandLess : expandMore);
-
-    const iconHTML = icon ? unsafeHTML(sanitizeSVG(icon)) : html``;
     return html`
-      <article class="m-accordion">
-        <div class="m-accordion__container">
-          <div
-            class="${classMap(titleContainerClasses)}"
-            @click="${this.toggleAccordion}"
+      <div class="m-accordion">
+        <div
+          class="${classMap(titleContainerClasses)}"
+          aria-role="${this.level ? 'heading' : ''}"
+          aria-level="${this.level ? this.level : ''}"
+          @click="${this.toggleAccordion}"
+        >
+          <button
+            id="${this.refId}"
+            class="m-accordion__button"
+            aria-expanded="${this.open}"
+            aria-controls="${this.refId}-content"
+            ?disabled="${this.disabled}"
           >
-            <div class="m-accordion__title-container-title">
-              <div class="${iconClass}">${iconHTML}</div>
-              <span class="${classMap(titleClasses)}">${title}</span>
+            <div class="m-accordion__button-flex-wrapper">
+              <div class="m-accordion__button-title-wrapper">
+                ${icon
+                  ? html`<div class="m-accordion__icon">
+                      ${unsafeHTML(sanitizeSVG(icon))}
+                    </div>`
+                  : html``}
+                <span class="${classMap(titleClasses)}">${title}</span>
+              </div>
+              <div class="m-accordion__icon-expand">
+                ${unsafeHTML(open ? expandLess : expandMore)}
+              </div>
             </div>
-            <div class="m-accordion__title-container--closed">
-              ${statusIcon}
-            </div>
-          </div>
-          <div
-            class="m-accordion__content ${open
-              ? 'm-accordion__content--open'
-              : ''}"
-          >
-            <slot></slot>
-          </div>
+          </button>
         </div>
-      </article>
+        <div
+          id="${this.refId}-content"
+          role="region"
+          aria-labelledby="${this.refId}"
+          class="m-accordion__content ${open
```

Comment:
missed opportunity to use classMap, which would make this less brittle- please adapt

Reply:
General rule for me would be: once you use classMap in a component, use it  consistently wherever a class list is not completely static!